instrumentation/nacos-client-2.0.3/javaagent/src/test/java/com/alibaba/nacos/shaded/com/google/errorprone/annotations/DoNotMock.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.alibaba.nacos.shaded.com.google.errorprone.annotations;
+
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+@Target(TYPE)
+@Retention(RUNTIME)
+@Documented
+public @interface DoNotMock {
```

Comment:
is this used anywhere?

Reply:
The class added here is to resolve a test compilation error in the `nacos-client` version 2.0.3 jar:

> Compilation failed; see the compiler output below.  
Error: Warning found, but specified -Werror  
G:\temp\.gradle\caches\modules-2\files-2.1\com.alibaba.nacos\nacos-client\2.0.3\a73c03b466a1f1565ee2b0e459c43046d6c8c15b\nacos-client-2.0.3.jar(/com/alibaba/nacos/shaded/com/google/common/util/concurrent/ListenableFuture.class): Warning: Unable to find class 'DoNotMock' annotation method 'value()': Can't find it com.alibaba.nacos.shaded.com.google.errorprone.annotations.DoNotMock class file.

If you choose to use version 2.0.0, this error will not occur. I will make changes here.